atd-toolbox/data_model/python/debug_record_mismatched/debug_lat_lon_diff.sql

```diff
@@ -0,0 +1,41 @@
+-- after populating VZ differences
+-- calculate the difference between CRIS lat/lon and edits lat lon
+WITH joined AS (
+    SELECT
+        cris.id,
+        ST_SETSRID(ST_MAKEPOINT(
+            cris.longitude,
+            cris.latitude
+        ),
+        4326)::geography AS cris_pt,
+        ST_SETSRID(ST_MAKEPOINT(
+            edits.longitude,
+            edits.latitude
+        ),
+        4326)::geography AS edits_pt
+    FROM
+        crashes_cris AS cris
+    LEFT JOIN crashes_edits AS edits ON cris.id = edits.id
+    WHERE
+        cris.latitude IS NOT NULL
+        AND cris.longitude IS NOT NULL
+        AND edits.latitude IS NOT NULL
+        AND edits.longitude IS NOT NULL
+),
+
+dists AS (
+    SELECT
+        id,
+        ST_DISTANCE(
+            cris_pt,
+            edits_pt
+        ) AS dist
+    FROM
+        joined
+)
+
+SELECT *
+FROM
+    dists
+WHERE
+    dist < 10;
```

Comment:
why where dist < 10?

Reply:
this was just me noodling to try an understand why we have tens of thousands of edits on latitude/longitude. you can adjust the `dist` (which is in meters) to get a sense how minor some of these edits are. 

it seems that the majority of these edits are actually the result of CRIS changing their geocoding process, or reference layers, over time. it's mostly harmless, but it might be nice to discard edits within a certain threshold, e.g 1 meter. but it's not a priority.